met-cron/migrations/versions/e817bbaef67e_add_sentiment_to_textarea.py

```diff
@@ -0,0 +1,26 @@
+"""add_sentiment_to_textarea
+
+Revision ID: e817bbaef67e
+Revises: fe0a86c0d458
+Create Date: 2022-09-15 16:57:11.296144
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e817bbaef67e'
+down_revision = 'fe0a86c0d458'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('response_type_textarea', sa.Column('sentiment', sa.String(length=50), nullable=True))
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
When downgrading he version ideally we want to revert the changes made, in this case drop the column